addons/escoria-core/game/core-scripts/esc_animation_player.gd

```diff
@@ -59,7 +59,7 @@ func _ready() -> void:
 # **Returns** the currently playing animation name
 func get_animation() -> String:
 	if _is_animation_player:
-		return _animation_player.current_animation
+		return _animation_player.assigned_animation
```

Comment:
Should the docstring for this method be updated to reflect that it returns the assigned animation as opposed to the current animation?

Reply:
No, these are actually the same ; they're just named differently between `AnimationPlayer` and `AnimatedSprite` nodes.